src/Code/Viewer.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Box, IBox } from '../Box';
+import { codeStyles } from './styles';
+import { mapWithDepth } from './utils/astToReact';
+import { parseCode } from './utils/parseCode';
+
+export interface IViewer extends IBox {
+  value: string;
+  language?: string;
+  // todo: implement me
+  // showLineNumbers?: boolean;
```

Comment:
@marbemac do we need showLineNumbers right now? Happy to implement if so.

Reply:
Yes, if it is not difficult. A number of our docs users use this feature right now.

Reply:
If it takes longer than an hour or two we can make a separate story for it.

Reply:
Nah, should be straightforward (< 1hr) :)